flink-streaming-java/src/test/java/org/apache/flink/streaming/api/SourceFunctionTest.java

```diff
@@ -52,12 +52,4 @@ public void fromCollectionTest() throws Exception {
 						Arrays.asList(1, 2, 3))));
 		assertEquals(expectedList, actualList);
 	}
-
```

Comment:
I think it would be good to have a snapshot/restore test for this source, verifying that we see all the expected elements (no matter the order).